atomics/include/desul/atomics/Fetch_Op_GCC.hpp

```diff
@@ -16,31 +16,31 @@ namespace desul {
 namespace Impl {
 
 // clang-format off
-#define DESUL_IMPL_GCC_HOST_ATOMIC_FETCH_OP_INTEGRAL_ORDER_SCOPE(OP, MEMORY_ORDER, MEMORY_SCOPE)                                 \
+#define DESUL_IMPL_GCC_HOST_ATOMIC_FETCH_OP_INTEGRAL_ORDER_SCOPE(FETCH_OP, OP_FETCH, MEMORY_ORDER, MEMORY_SCOPE)                                 \
   template <class T>                                                                                                          \
```

Comment:
Same question as for hip and sycl here. Presumably this will not be in the MSVC code path.